CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/DatabaseRequestHandler.swift

```diff
@@ -0,0 +1,302 @@
+//
+//  DatabaseRequestHandler.swift
+//  CBLTestServer-iOS
+//
+//  Created by Raghu Sarangapani on 12/20/17.
+//  Copyright © 2017 Raghu Sarangapani. All rights reserved.
+//
+
+import Foundation
+import CouchbaseLiteSwift
+
+
+public class DatabaseRequestHandler {
+    public static let VOID: String? = nil
+    fileprivate var _pushPullReplListener:NSObjectProtocol?
+    
+    public func handleRequest(method: String, args: Args) throws -> Any? {
+        switch method {
+
+        //////////////
+        // Database //
+        //////////////
+    
+        case "database_create":
+            let name: String! = args.get(name: "name")
+            let dbConfig: DatabaseConfiguration? = args.get(name: "config")
+            do {
+                if dbConfig != nil {
+                    return try Database(name: name!, config: dbConfig!)
+                } else {
+                    return try Database(name: name!)
+                }
+            } catch {
+                return error.localizedDescription
+            }
+            
+        case "database_close":
+            let database: Database = args.get(name:"database")!
+            try database.close()
+
+        case "database_getPath":
+            let database: Database = (args.get(name:"database"))!
+
+            return database.path
+
+        case "database_delete":
+            let database: Database = (args.get(name:"database"))!
+            let document: Document = (args.get(name:"document"))!
+
+            do {
+                try database.deleteDocument(document)
+            } catch {
+                return error
+            }
+            
+        case "database_deleteBulkDocs":
+            let database: Database = args.get(name:"database")!
+            let doc_ids: Array<String> = args.get(name: "doc_ids")!
+
+            try database.inBatch {
+                for id in doc_ids {
+                    let document: Document = database.document(withID: id)!
+                    do {
+                        try! database.deleteDocument(document)
+                    }
+                }
+            }
+
+        case "database_deleteDB":
+            let database: Database = args.get(name:"database")!
+            
+            try database.delete()
+
+        case "database_exists":
```

Comment:
In android, the directory parameter is not option. Do we have a confirmation about it, for iOS?

Reply:
Not sure what you mean by confirmation, it is optional in swift - https://github.com/couchbase/couchbase-lite-ios/blob/master/Swift/Database.swift#L327